acceptance/tests/mesh-gateway/mesh_gateway_test.go

```diff
@@ -28,6 +28,7 @@ func TestMeshGatewayDefault(t *testing.T) {
 
 	primaryHelmValues := map[string]string{
 		"global.datacenter":                        "dc1",
+		"global.image":                             "thisisnotashwin/consul@sha256:477091fe84cde79a68a37cc9cc69fb7a5ab35e647a0f5f2632451ace5ecc5e7c",
```

Comment:
Should we have a global constant `ConsulImage` in the acceptance framework that we set? Might be convenient for these kinds of cross-cutting changes in the future.

Reply:
i think we do have one in the tests. i use this flag if I want to tinker with the image on only one of the tests but I have removed it here.